modules/microshift-backing-up-manually.adoc

```diff
@@ -0,0 +1,63 @@
+//Module included in the following assemblies:
+//
+// * microshift_updating/microshift-update-options.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-backing-up-manually_{context}"]
+= Backing up {product-title} data manually
+
+You can back up {product-title} data manually at any time. Data backups are advised prior to any system updates. Backups are created in the `/var/lib/microshift-backups` directory by default. You can also specify a directory by adding custom parameters to the required backup command.
+
+[NOTE]
+====
+On `rpm-ostree` systems, {product-title} creates an automatic backup on every start and restores data automatically.
+====
+
+.Prerequisites
+* You have root access to the host.
+* {product-title} is stopped.
+
+.Procedure
+. Manually create a backup using the default name and parent directory, `/var/lib/microshift-backups/<default-backup-name>`, by running the following command:
```

Comment:
```suggestion
. Manually create a backup that uses the default name and parent directory, `/var/lib/microshift-backups/<default-backup-name>`, by running the following command:
```
For clarity, replace "using" with either "that uses" or "by using", depending on your intended meaning.